internal/models/users/users.go

```diff
@@ -237,6 +243,46 @@ func DeleteUserByID(db *sql.DB, userID uuid.UUID) error {
 	return err
 }
 
+// Returns a list of all users in the database, with the data needed for the user administration table.
+func GetUsersTableRows(db *sql.DB, page int) ([]UserTableRow, error) {
+	const pageSize = 50
+	var offset = pageSize * (page - 1)
+
+	if offset < 0 {
+		return nil, errors.New("page number must be greater than 0")
+	}
+
+	rows, err := db.Query(`
+	SELECT
+		CONCAT(username_adjective, ' ', username_noun) AS username, SUM(points_awarded) AS total_points
+		FROM users LEFT JOIN user_answers
+			ON users.id = user_answers.user_id
+		GROUP BY users.id
```

Comment:
points_awarded used here is from user_answers table, this is depricated. we went over to using views with sql function to count points. they are not stored here. the column in table is left for now not to break ranking queries, Jakob is supposed to look into updating the queries to use the views.

anywa, this PR is user administration or username administration? i am confused

Reply:
unless it will be used for admin leaderboard this should be removed

Reply:
unused code, removed